src/ucs/memory/rcache.c

```diff
@@ -240,6 +302,11 @@ static void ucs_mem_region_destroy_internal(ucs_rcache_t *rcache,
         }
     }
 
+    if (ucs_rcache_is_pfn_enabled() &&
+        (ucs_global_opts.rcache_pfn_max_pages > 1)) {
+        free(ucs_rcache_region_pfn_ptr(region));
```

Comment:
can we avoid using malloc/free in this flow?

Reply:
ucs_free

Reply:
yep, fixed